main.cpp

```diff
@@ -331,25 +399,29 @@ int handle_args(int argc, char** argv) {
 
         if (vm.count("telemetry")) {
             logReader->add_source(new MetadataReader(*logReader, vm["telemetry"].as<string>()));
-            currentState.hasMetadataSource = true;
+            newState.hasMetadataSource = true;
+            cout << "Adding Telemetry source..." << endl;
         }
 
         if (vm.count("addr") && vm.count("port")) {
             logReader->add_source(new MetadataReader(*logReader, vm["addr"].as<string>(), vm["port"].as<string>()));
             newState.hasMetadataSource = true;
+            cout << "Adding Telemetry source..." << endl;
         }
 
 #ifdef HAS_DECKLINK
         if (vm.count("decklink")) {
-            importer.add_source(new DeckLinkImport(logReader), 500);
+            importer.add_source(new DeckLinkImport(logReader, goProRect), 500);
```

Comment:
Yes it should be configurable via CLI, but that would require having some notion of what a camera is from the perspective of the CLI, which realistically requires storing camera information in a file that can be loaded at runtime and specifying the appropriate camera when adding an image source. 
So for now, I thought we could just hardcode what we're going to be using.